apps/src/aiDifferentiation/AiDiffBotMessageFooter.tsx

```diff
@@ -34,26 +38,45 @@ const AiDiffBotMessageFooter: React.FC<Props> = ({message}) => {
             } else {
               copyToClipboard(message.chatMessageText);
             }
+            setCopyTimeout(true);
+            setTimeout(() => setCopyTimeout(false), CONFIRM_TIMEOUT_MS);
           }}
           color="white"
           size="xs"
           isIconOnly
-          icon={{iconStyle: 'regular', iconName: 'copy'}}
+          icon={{
+            iconStyle: 'regular',
+            iconName: copyTimeout ? 'check' : 'copy',
+          }}
           type="primary"
-          className={style.messageFeedbackButton}
+          className={
+            copyTimeout
+              ? style.messageFeedbackConfirm
+              : style.messageFeedbackButton
+          }
         />
         <PDFDownloadLink
           document={<AiDiffPdf messages={[message]} />}
           fileName="ai_differentiation_message.pdf"
         >
           <Button
-            onClick={() => {}}
+            onClick={() => {
```

Comment:
Maybe consider disabling the download action on this button when the checkmark is displayed, to prevent a double-click triggering a double download?

Reply:
done